contracts/test/mocks/lib/token/TokenBalanceOfAllowanceMissingReturnMock.sol

```diff
@@ -0,0 +1,106 @@
+// Non-standards compliant token that is missing return values for
+// `allowance()` and `balanceOf()`.
+// Modified from https://github.com/OpenZeppelin/openzeppelin-solidity/blob/a9f910d34f0ab33a1ae5e714f69f9596a02b4d91/contracts/token/ERC20/StandardToken.sol
+
+pragma solidity 0.4.24;
+
+import "../../../../lib/math/SafeMath.sol";
+
+
+contract TokenBalanceOfAllowanceReturnMissingMock {
+    using SafeMath for uint256;
+    mapping (address => uint256) private balances;
+    mapping (address => mapping (address => uint256)) private allowed;
+    uint256 private totalSupply_;
+    bool private allowTransfer_;
+
+    event Approval(address indexed owner, address indexed spender, uint256 value);
+    event Transfer(address indexed from, address indexed to, uint256 value);
+
+    // Allow us to set the inital balance for an account on construction
+    constructor(address initialAccount, uint256 initialBalance) public {
+        balances[initialAccount] = initialBalance;
+        totalSupply_ = initialBalance;
+        allowTransfer_ = true;
+    }
+
+    function totalSupply() public view returns (uint256) { return totalSupply_; }
```

Comment:
Now that I'm thinking of... shouldn't we provide a static version of `totalSupply` in `SafeERC20`?

Reply:
JIC https://github.com/aragon/aragonOS/pull/543

Reply:
Makes sense; we didn't have a use case for it before but it doesn't hurt to include it :).